shell/assets/styles/themes/_dark.scss

```diff
@@ -31,6 +31,22 @@
 
   $drag-over: #DCDEE7;
 
+  // Status colors Yaml
```

Comment:
Since we are already (reasonably) squashing the name into an acronym (`cm`), could we please write a bit more extended comment here.
Something like "Colors customization for CodeMirror code highlight in Yaml mode" would be more insightful on why suddenly a block of custom colors out of standards. We can add this once on top, so we don't have to repete it twice, although it has to be added also for the light colors.